models/banner.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    message: Joi
+        .string()
+        .max(512)
+        .description('Body of banner to display')
+        .example('Due to planned upgrade of Kubernetes, Screwdriver will be down')
+};
+
+module.exports = {
+    /**
+     * All the available properties of Banner
+     *
+     * @property base
+     * @type {Joi}
+     */
+    base: Joi.object(MODEL).label('Banner'),
+
+    /**
+     * Properties for User that will be passed during a CREATE request
+     *
+     * @property create
+     * @type {Joi}
+     */
+    create: Joi.object(mutate(MODEL, [
```

Comment:
I imagine you will have an `update` too right? (for people to update old banner)
And maybe `get or `list`, depending on what you'll need it on the API side.

Reply:
I think Min and I were thinking just an update (`PUT`) and delete for the first pass. https://github.com/screwdriver-cd/screwdriver/issues/966#issuecomment-378743186

Reply:
I think we need get and list for the UI to render it